src/functionalTest/java/uk/gov/hmcts/reform/hmi/vh/PostCloneVideoHearingTest.java

```diff
@@ -34,8 +35,11 @@ void setup() {
 
     /**
      * Test with a valid hearing id and a valid set of headers, response should return 204.
+     * This test is failing. We have asked VH the reason for it. This needs to be enable once
```

Comment:
Are we sure this failure is not related to this PR, and if so are we able to leave it enabled or does it block the build? I'd prefer to keep this enabled and it fail, as a reminder for something that needs fixing if we can